types/src/v18/blockchain/mod.rs

```diff
@@ -70,3 +70,52 @@ pub struct MempoolEntry {
     #[serde(rename = "bip125-replaceable")]
     pub bip125_replaceable: bool,
 }
+
+/// Result of JSON-RPC method `scantxoutset`.
+///
+/// > scantxoutset "action" ( [scanobjects,...] )
+/// >
+/// > Arguments:
+/// > 1. "action"                       (string, required) The action to execute
+/// > 2. "scanobjects"                  (array, required) Array of scan objects
+#[derive(Deserialize, Debug, Clone, PartialEq)]
+#[serde(untagged)]
+pub enum ScanTxOutSet {
+    /// Returns after scan completes
+    Start(ScanTxOutSetStart),
+    /// True (scan will be aborted), False (no scan to abort)
+    Abort(bool),
+    /// Scan in progress or Completed
+    Status(Option<ScanTxOutSetStatus>),
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ScanTxOutSetStart {
```

Comment:
This looks wrong. Perhaps add `#[serde(deny_unknown_fields)]` to all your structs and that will help find the bugs.

Reply:
Yeah I added it and tested, there are two missing fields, `success` and `searched_items`.